src/v2/cookbook/practical-use-of-scoped-slots.md

```diff
@@ -1,23 +1,23 @@
 ---
-title: Practical use of scoped slots with GoogleMaps
+title: Uso práctico de scoped slots con GoogleMaps
 type: cookbook
 order: 14
 ---
 
-## Base Example
+## Ejemplo Base
 
-There are situations when you want the template inside the slot to be able to access data from the child component that is responsible for rendering the slot content. This is particularly useful when you need freedom in creating custom templates that use the child component's data properties. That is a typical use case for scoped slots.
+Hay situaciones en la que usted querrá que el _template_ dentro del _slot_ acceda a los datos del componente hijo que es responsable de renderizar el contenido del _slot_. Esto es particularmente útil cuando usted necesita libertad al crear _templates_ personalizados que utilizen los datos y propiedades del componente hijo. Este es un caso típico para el uso de _scoped slots_.
 
-Imagine a component that configures and prepares an external API to be used in another component, but is not tightly coupled with any specific template. Such a component could then be reused in multiple places rendering different templates but using the same base object with specific API.
+Imagine una componente que configure y prepara una API externa para ser usada en otra componente, pero no esta fuertemente acoplada a un _template_ específico. Tal componente puede ser reusado en muchos lugares renderizando diferentes _templates_ pero usando el mismo objeto con la API específica.
```

Comment:
Please check this sentence `Imagine una componente que configure y prepara una API externa para ser usada en otra componente, pero no esta fuertemente acoplada a un _template_ específico.`